packages/components/src/Combobox/components/ComboboxContent/ComboboxContentSearch/ComboboxContentSearch.css

```diff
@@ -42,5 +42,9 @@
 }
 
 .clearSearch:focus {
+  outline: transparent;
```

Comment:
Resolves the extra outline we get from the browser
![image](https://github.com/GetJobber/atlantis/assets/39704901/4574f964-89f6-4358-a6b6-19a35a0b87e9)